spinn_machine/tags/iptag.py

```diff
@@ -61,41 +64,41 @@ def __init__(
         self._destination_y = destination_y
 
     @property
-    def ip_address(self):
+    def ip_address(self) -> Optional[str]:
```

Comment:
This is extra odd, since it is not optional in the constructor, so maybe not here too?

Reply:
agreed fixed.